### DISCUSSION TITLE
[Visual] Add custom Backround Colour for Widget

### DISCUSSION THREAD

**Discussion by galaxyrun** [breezy-weather/breezy-weather#1242] [Ideas]
### Describe your idea

It would be great if you could adjust the background color using a hex code, instead of just choosing between dark or light.

### Acknowledgements

- [X] I have searched the [existing issues](https://github.com/breezy-weather/breezy-weather/issues) and [existing discussions](https://github.com/breezy-weather/breezy-weather/discussions/categories/ideas) and this is a new ticket, **NOT** a duplicate or related to another open OR closed issue or discussion.
- [X] The idea I’m suggesting is not in the [list of features that will not be implemented](https://github.com/breezy-weather/breezy-weather/blob/main/README.md#features-that-will-not-be-implemented).
- [X] I have updated the app to **[version 5.2.6 or later](https://github.com/breezy-weather/breezy-weather/releases/latest)**. Even if I’m getting updates from F-Droid default repo, I don’t need to uninstall/reinstall to update.
- [X] I wrote my request in English, and any screenshot I attached are also in English (language can be changed from Settings > Appearance). I can write in my native language under the English text, if necessary.
- [X] I understand that this app is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY, so I may not get a positive reply to my request, if any at all.

**Comment by papjul** [comment]
Not exactly what you asked but we have plans to put the Material You color as background color during the widget overhaul #937. It's already available for 2 widgets. If you can define it in your Android settings, you can technically put any background color.
